src/ProductConstructionService/ProductConstructionService.Api/Program.cs

```diff
@@ -54,5 +66,13 @@
     var queueClient = queueServiceClient.GetQueueClient(app.Configuration[QueueConfiguration.JobQueueConfigurationKey]);
     await queueClient.CreateIfNotExistsAsync();
 }
+// When running in Azure add the vmrCloned health check
+else
+{
+    app.MapHealthChecks("/vmrReady", new HealthCheckOptions
```

Comment:
So if we had other checks would they go under one endpoint/readiness check? Or would they get their own?

Maybe we can just have `/alive` and `/ready` and make `/ready` unify all we need?

Reply:
This endpoint will only check the new `VmrReadyHealthCheck` because of the tag we gave it. Besides this one, we have an `/alive` endpoint, which just tests the responsiveness of the service, and a `/health` endpoint, which runs all tests. Maybe we can use that one, and not add a new one, true

Reply:
I removed this and changed the probe to target `/health` since it will run all HealthChecks we have